portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase

```diff
@@ -113,6 +113,82 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-10257 | Test Summary: Assert total amount of views in all cards"
+	@ignore = "true"
+	@priority = "4"
+	test AssertTotalAmountOfViewsInAllCards {
+		ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		task ("Add a new user in DXP") {
+			JSONUser.addUser(
+				userEmailAddress = "ac@liferay.com",
+				userFirstName = "ac",
+				userLastName = "ac",
+				userScreenName = "ac");
+		}
+
+		task ("Sync Contact Data") {
+			ACDXPSettings.syncNewContactData();
+		}
+
+		task ("Login new user and view page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "ac@liferay.com",
+				userLoginFullName = "ac ac");
+		}
+
+		task ("Interation with page") {
+			ACUtils.navigateToSitePageWithUser(
+				pageName = "AC Page",
+				siteName = "Site Name");
+
+			ACUtils.closeAllSessionsAndWait();
+		}
+
+		task ("AC launch") {
+			ACUtils.launchAC();
+
+			ACProperties.switchProperty(propertyName = "${propertyName}");
+		}
+
+		task ("Asset views for 24 hours") {
+			ACNavigation.goToSites();
+
+			ACNavigation.switchTab(tabName = "Pages");
+
+			ACTimeFilter.clickTimeFilterButton();
+
+			ACTimeFilter.setLast24Hours();
+
+			ACNavigation.openItem(itemName = "AC Page - Site Name - ${dataSourceName}");
+		}
+
+		task ("Get views value") {
+			var current = ACSettings.getCurrentNumber(metricName = "Views");
+
+			ACTimeFilter.clickTimeFilterButton();
+
+			ACTimeFilter.setLast24Hours();
```

Comment:
Hey @calvinctvo, I think all the graphs will already have in the last 24hour filter because it was the filter you defined when [selecting the page](https://github.com/liferay-ac/liferay-portal/pull/9/files#diff-ec1c3bef7cea0f482aa4c48aa2240f4a914cbd92ceed090bdc3ff826f4642ebaR159-R161). What do you think?